### PR TITLE
Add open graph meta tags

### PR DESCRIPTION
**Pull request by brunobesson**

### CODE REVIEW

File: src/js/vue-plugins/gettext-plugin.js [hunk 1]
```diff
@@ -177,6 +177,31 @@ export default function install(Vue) {
...
+            return 'it_IT';
+          case 'de':
+            return 'de_DE';
+          case 'es':
+            return 'es_ES';
+          case 'zh_CN':
+            return 'zh_CN';
+          default:
+            // eslint-disable-next-line no-console
+            console.error(`Unexpected langauage: ${lang}`);
```
Comment (AntoineMarnat) [inline src/js/vue-plugins/gettext-plugin.js:200]: typo ;)

Comment (brunobesson) [reply src/js/vue-plugins/gettext-plugin.js:200]: 🙂

File: src/js/vue-plugins/gettext-plugin.js [hunk 2]
```diff
@@ -172,7 +172,32 @@ export default function install(Vue) {
...
+      getIsoLanguageTerritory(lang) {
+        switch (lang) {
+          case 'fr':
+            return 'fr_FR';
+          case 'en':
+            return 'en_UK';
+          case 'ca':
+            return 'ca_ES';
+          case 'eu':
+            return 'eu_E';
```
Comment (ataraxido) [inline src/js/vue-plugins/gettext-plugin.js:189]: typo ? does not exist

File: src/views/document/utils/document-view-mixin.js [hunk 3]
```diff
@@ -67,6 +68,12 @@ export default {
         },
       ];
     },
+    meta: function () {
+      if (!this.document) {
+        return null;
+      }
+      return this.documentOpenGraph();
```
Comment (ataraxido) [inline src/views/document/utils/document-view-mixin.js:75]:
```suggestion
      if (this.document) {
        return this.documentOpenGraph();
      }
      return null;
```

File: src/js/vue-plugins/gettext-plugin.js (hunk 2 above)
Comment (brunobesson) [reply src/js/vue-plugins/gettext-plugin.js]: 👍

File: src/views/document/utils/document-view-mixin.js (hunk 3 above)
Comment (brunobesson) [reply src/views/document/utils/document-view-mixin.js:75]: I'd rather keep the logic the other way: first return fast with excluding conditions, then expand the main logic (although it's simple here)

**Pull request merged**